app/views/api/v0/profile_images/show.json.jbuilder

```diff
@@ -0,0 +1,4 @@
+json.type_of "profile_image"
+
+json.profile_image Images::Profile.call(@profile_image_url, length: 640)
+json.profile_image_90 Images::Profile.call(@profile_image_url, length: 90)
```

Comment:
I read your doubts in the description but I think the serialization is simple enough to be effective, if we need more versions of the image we can add them but this closely resembles what we send back with the user or organization representations in their own partials.

I would add a key that should help the client: is this the image of a user or an org? 

Maybe:

```json
{
  "type_of": "profile_image",
  "image_of: "user"
  ...
}
```

what do you think?

Reply:
Yup I tinkered a bit with the idea of embedding the owner somehow on the response:

```json
{
  ...
  "owner": {
     "type_of": "user" | "organization",
      ...
  }
}
```

But given the use case (simply retrieving the profile image), it seemed an overkill tbh. I also tinkered with the possibility of trying to model this using [OpenAPI Links](https://swagger.io/docs/specification/links/), but I couldn't find a way to do so. 😄 

Anyway I agree and I ended up implementing your suggestion. 👍